stencil.config.ts

```diff
@@ -24,5 +24,5 @@ export const config: Config = {
       serviceWorker: null, // disable service workers
     },
   ],
-  globalStyle: "src/globals/global-styles.css"
+  globalStyle: "src/global.css"
```

Comment:
this becomes useless if the `global.css` file is left empty

Reply:
thi is useful considering the commenti right above